tests/components/brother/test_config_flow.py

```diff
@@ -0,0 +1,127 @@
+"""Define tests for the Brother Printer config flow."""
+import json
+
+from asynctest import patch
+from brother import SnmpError, UnsupportedModel
+
+from homeassistant import data_entry_flow
+from homeassistant.components.brother import config_flow
+from homeassistant.components.brother.const import DOMAIN
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_TYPE
+
+from tests.common import load_fixture
+
+CONFIG = {
+    CONF_HOST: "localhost",
+    CONF_NAME: "Printer",
+    CONF_TYPE: "laser",
+}
+
+
+async def test_show_form(hass):
+    """Test that the form is served with no input."""
+    flow = config_flow.BrotherConfigFlow()
+    flow.hass = hass
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": "user"}
+    )
+
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+    assert result["step_id"] == "user"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4770368)